srcpkgs/ROCm-SMI-LIB/patches/0001-cmake-system-processor.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index b3eff5d..defc6b2 100755
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -76,7 +76,7 @@ endif()
+ ## Compiler flags
+ set(CMAKE_CXX_FLAGS
+  "${CMAKE_CXX_FLAGS} -Wall -Wextra -fno-rtti")
+-if (${CMAKE_HOST_SYSTEM_PROCESSOR} STREQUAL "x86_64")
++if (${CMAKE_SYSTEM_PROCESSOR} STREQUAL "x86_64")
```

Comment:
Fixes cross compiling on arm

Also fixes cross compiling on armv* in the extremely unlikely event that anybody will want to use this package there.

Made a PR upstream: https://github.com/ROCm/rocm_smi_lib/pull/158